### PR TITLE
Rdb initial validation

### PR DESCRIPTION
**Pull request by dideek**

### CODE REVIEW

File: grisera_api/arrangement/arrangement_service_relational.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def get_arrangement(self, arrangement_id: Union[int, str], depth: int = 0, sourc
 
         if depth > 0 and source != Collections.ACTIVITY_EXECUTION:
                 arrangement_dict["activity_executions"] = activity_execution_service.get_multiple_with_foreign_id(arrangement_id, depth - 1, self.table_name)
-
+        print(arrangement_dict)
```
Comment (Noxxxxious) [inline grisera_api/arrangement/arrangement_service_relational.py]: Print debugging to remove

File: grisera_api/activity_execution/activity_execution_service_relational.py [hunk 2]
```diff
@@ -48,18 +48,23 @@ def get_activity_execution(self, activity_execution_id: Union[int, str], depth:
...
                 activity_execution["participations"] = participation_service_relational.get_multiple_with_foreign_id(
                     activity_execution["id"], depth - 1, self.table_name)
             if source != Collections.EXPERIMENT and source != Collections.SCENARIO:
                 activity_execution["experiments"] = scenario_service_relational.get_experiment_by_activity_execution(
                     activity_execution["id"], depth - 1, self.table_name)
             if source != Collections.ARRANGEMENT:
-                activity_execution["arrangements"] = arrangement_service_relational.get_arrangement(
+                arrangement = arrangement_service_relational.get_arrangement(
                     activity_execution["arrangement_id"], depth - 1, self.table_name)
+                if type(arrangement) is not NotFoundByIdModel:
```
Comment (Noxxxxious) [inline grisera_api/activity_execution/activity_execution_service_relational.py:66]: Impossible situation, activity execution record can't possibly store non-existent foreign key

File: grisera_api/activity_execution/activity_execution_service_relational.py [hunk 3]
```diff
@@ -106,7 +111,7 @@ def update_activity_execution_relationships(self, activity_execution_id: Union[i
         return ActivityExecutionOut(**put_result["records"])
 
     def get_multiple_with_foreign_id(self, foreign_id: Union[int, str], depth: int = 0, source: str = ""):
-        activity_executions = self.rdb_api_service.get_records_with_foreign_id(self.table_name, source+"_id",foreign_id)
+        activity_executions = self.rdb_api_service.get_records_with_foreign_id(self.table_name, source+"_id",foreign_id)["records"]
```
Comment (Noxxxxious) [inline grisera_api/activity_execution/activity_execution_service_relational.py:114]: Add error handling via "errors" key check

File: grisera_api/activity_execution/activity_execution_service_relational.py [hunk 4]
```diff
@@ -132,5 +137,5 @@ def get_multiple_with_foreign_id(self, foreign_id: Union[int, str], depth: int =
                 for activity_execution in activity_executions:
                     activity_execution["arrangements"] = arrangement_service_relational.get_arrangement(
                         activity_execution["arrangement_id"], depth - 1, self.table_name)
-        return ActivityExecutionsOut(activity_executions=activity_executions)
-    
+        return [ActivityExecutionOut(**execution) for execution in activity_executions]
```
Comment (Noxxxxious) [inline grisera_api/activity_execution/activity_execution_service_relational.py:140]: Nit: could just return activity_executions

File: grisera_api/activity_execution/activity_execution_service_relational.py (hunk 2 above)
Comment (dideek) [reply grisera_api/activity_execution/activity_execution_service_relational.py:66]: Someone can add a record with no foreign key

File: grisera_api/activity_execution/activity_execution_service_relational.py (hunk 4 above)
Comment (dideek) [reply grisera_api/activity_execution/activity_execution_service_relational.py:140]: this cannot produce an error

**Pull request merged**